server/routes/public.js

```diff
@@ -38,6 +89,9 @@ router.get('/create-user', async (req, res) => {
     const phone = req.query.phone;
 
     const user = await accountHandler.createUserAccount(firstName, lastName, type, studentID, email, password, phone);
+>>>>>>> master
```

Comment:
Hi Nancy,
I would suggest removing lines 92 to 94 as these lines will interfere with the code execution. I think they were left behind as a result of resolving the merge conflicts.

Reply:
Hi Amy,
Good catch! I have removed the lines indicating merging changes!